### ISSUE TITLE
How the Web works

### ISSUE THREAD

**Issue by zhouzhenyuan** [zhouzhenyuan/automatic-waddle#5]
### Clients and servers

![](https://mdn.mozillademos.org/files/8973/Client-server.jpg)

### The other parts of the toolbox

- **TCP/IP**: Transmission Control Protocol and Internet Protocol are communication protocols that define how data should travel across the web.
- **DNS**: Domain Name Servers.
- **HTTP**: Hypertext Transfer Protocol is an application protocol that defines a language for clients and servers to speak to each other.

### Packets explained

Basically, when data is sent across the web, it is sent as thousands of small chunks, so that many different web users can download the same website at the same time.
